<launch>
    <arg name="outside" default="false"/>
    <arg name="cameras" default="false"/>

    <include file="$(find mrover)/launch/jetson_base.launch">
        <arg name="cameras" value="$(arg cameras)"/>
        <arg name="localization" value="$(arg outside)"/>
    </include>

    <!-- Launch the ZED -->
     <!-- This is after the percep nodelet manager which was created in the jetson base -->
    <include file="$(find mrover)/launch/zed.launch">
        <arg name="zed_model" value="zed"/>
        <arg name="serial_number" value="14458"/>
      </include>

    <!-- Launch Autonomy -->
    <include file="$(find mrover)/launch/autonomy.launch">
        <arg name="tag_detector" value="true"/>
        <arg name="object_detector" value="false"/>
        <arg name="cost_map" value="false"/>
        <arg name="localization" value="true"/>
    </include>

    <!-- Run the IMU Driver -->
    <node name="imu_driver" pkg="mrover" type="imu_driver.py"
        respawn="true" respawn_delay="2"
        output="screen"/>

    <!-- Launch the light detector -->
    <include file="$(find mrover)/launch/light_detector.launch">
        <arg name="sim" value="false"/>
    </include>

</launch>
